packages/react-components/react-field/src/components/FieldMessage/FieldMessage.types.ts

```diff
@@ -0,0 +1,26 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+/**
+ * FieldMessage Slots
+ */
+export type FieldMessageSlots = {
+  root: NonNullable<Slot<'div'>>;
+
+  /**
+   * Icon displayed before the message text. The default icon depends on the validationState.
```

Comment:
I realize this is more or less what it said before, but it might be nice to list the default icons per validation state, or maybe just mention that `neutral` defaults to no icon. Doesn't need to be a blocker to this PR though, since it's not really changed here.